docs/guides/usage/tracing.livemd

```diff
@@ -165,14 +210,13 @@ Integer.parse("1")
 <!-- livebook:{"output":true} -->
 
 ```
-Matcha.Trace: `Elixir.Integer.parse("1")` called on #PID<0.142.0>
-Recon tracer rate limit tripped.
+{1, ""}
 ```
 
 <!-- livebook:{"output":true} -->
 
 ```
-{1, ""}
+Matcha.Trace: `Elixir.Integer.parse("1")` called on #PID<0.135.0>
```

Comment:
I think part of these diffs are because each execution of the cell has a chance to return (`{1, ""}`) ***and*** receive trace messages in any order. I'd generally ignore diffs where that's the only thing happening, and pretend it just always shows the return before the trace. I'd accept manual editing of the result into a stand form?